web-common/src/test/java/edu/unc/lib/boxc/web/common/services/AccessCopiesServiceTest.java

```diff
@@ -364,6 +392,51 @@ public void hasViewableFilesImageWorkTest() {
         assertRequestedDatastreamFilter(DatastreamType.JP2_ACCESS_COPY);
     }
 
+    @Test
+    public void hasStreamingSoundTest() {
+        var mdObjectAudio = createAudioObject(ResourceType.Work);
+        hasPermissions(mdObjectAudio, true);
+
+        when(searchResultResponse.getResultList()).thenReturn(List.of(mdObjectAudio));
+        var audioObj = accessCopiesService.getStreamingChildren(mdObjectAudio, principals);
+        assertEquals("sound", audioObj.getStreamingType());
+        assertHasPopulatedFieldFilter(SearchFieldKey.STREAMING_TYPE);
+    }
+
+    @Test
+    public void hasStreamingVideoTest() {
+        var mdObjectVideo = createVideoObject(ResourceType.Work);
+        hasPermissions(mdObjectVideo, true);
+        when(searchResultResponse.getResultList()).thenReturn(List.of(mdObjectVideo));
+        var videoObj = accessCopiesService.getStreamingChildren(mdObjectVideo, principals);
+        assertEquals("video", videoObj.getStreamingType());
+        assertHasPopulatedFieldFilter(SearchFieldKey.STREAMING_TYPE);
+    }
+
+    @Test
+    public void doesNotHaveStreamingAudioTest() {
+        hasPermissions(mdObjectImg, true);
+        when(searchResultResponse.getResultCount()).thenReturn(0L);
+        assertNull(accessCopiesService.getStreamingChildren(mdObjectImg, principals));
+        assertHasPopulatedFieldFilter(SearchFieldKey.STREAMING_TYPE);
+    }
+
+    @Test
+    public void doesNotHaveStreamingVideoTest() {
+        hasPermissions(mdObjectImg, true);
+        when(searchResultResponse.getResultCount()).thenReturn(0L);
+        assertNull(accessCopiesService.getStreamingChildren(mdObjectImg, principals));
+        assertHasPopulatedFieldFilter(SearchFieldKey.STREAMING_TYPE);
+    }
```

Comment:
These two tests are identical, we should probably just keep one and name it "doesNotHaveStreamingTest"